src/shogun/kernel/ANOVAKernel.h

```diff
@@ -105,32 +86,21 @@ class ANOVAKernel: public DotKernel
 	 */
 	float64_t compute_rec2(int32_t idx_a, int32_t idx_b);
 
-	/** Casts the given kernel to CANOVAKernel.
-	 * @param kernel Kernel to cast. Must be CANOVAKernel. Might be NULL
-	 * @return casted CANOVAKernel object, NULL if input was NULL
+	/** Casts the given kernel to ANOVAKernel.
+	 * @param kernel Kernel to cast. Must be ANOVAKernel. Might be NULL
+	 * @return casted ANOVAKernel object, NULL if input was NULL
 	 */
 	static std::shared_ptr<ANOVAKernel> obtain_from_generic(const std::shared_ptr<Kernel>& kernel);
```

Comment:
I think these are not used anymore, so can be remove (see what happens if you do that)

Reply:
Makes sense, removed it.